### PR TITLE
system tools update

### PR DESCRIPTION
**Pull request by CvH**
changes are not yet runtime tested

- adds ytop
image

### CODE REVIEW

File: packages/addons/tools/system-tools/package.mk [hunk 1]
```diff
@@ -133,8 +133,8 @@ addon() {
     cp -P $(get_install_dir nmon)/usr/bin/nmon $ADDON_BUILD/$PKG_ADDON_ID/bin/
 
     # p7zip
-    cp -P $(get_install_dir p7zip)/usr/bin/{7z,7za,7z.so} $ADDON_BUILD/$PKG_ADDON_ID/bin
-    cp -PR $(get_install_dir p7zip)/usr/bin/Codecs $ADDON_BUILD/$PKG_ADDON_ID/bin
+    cp -P $(get_install_dir p7zip)/usr/bin/{7z,7za,7z.so} $ADDON_BUILD/$PKG_ADDON_ID/lib
+    cp -PR $(get_install_dir p7zip)/usr/bin/Codecs $ADDON_BUILD/$PKG_ADDON_ID/lib
```
Comment (HiassofT) [inline packages/addons/tools/system-tools/package.mk:138]: copy these to `$ADDON_BUILD/$PKG_ADDON_ID/lib/p7zip` and adjust the wrapper scripts and `mkdir -p` accordingly.

This is also how it's done in Debian/Ubuntu and will prevent 7z.so from being in LD_LIBRARY_PATH

Comment (CvH) [reply packages/addons/tools/system-tools/package.mk:139]: updated

Comment (HiassofT) [reply packages/addons/tools/system-tools/package.mk:139]: thanks, looks good now!

**Comment by CvH** [comment]
limited runtime test were okay 👍

**Pull request merged**